EIPS/Skinny_CREATE2.md

```diff
@@ -0,0 +1,16 @@
+```
+EIP: <to be assigned>
```

Comment:
Please number this 1014 and rename the file to eip-1014.md.

Reply:
0xff would work, but 0x01 would be not even theoretically possible to collide (on the preimage side)